src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/CourtService.kt

```diff
@@ -62,4 +72,66 @@ class CourtService(
         )
       }.toSet()
   }
+
+  fun createVideoLinkBooking(specification: VideoLinkBookingSpecification): Long {
```

Comment:
Should this be annotated @Transactional?

Reply:
Yes!